selvansgeo.py

```diff
@@ -293,9 +265,8 @@ def connectionsInit(self):
                 if checkdb.open():
                     self.credentialInstance.put(connectionInfo, user, pwd)
                 else:
-                    self.messageBar.pushMessage("Erreur",
-                                                str(u"Mauvais mot de passe"),
-                                                level=QgsMessageBar.CRITICAL)
+                    self.messageBar.pushCritical(
+                        "Erreur", str(u"Mauvais mot de passe"))
```

Comment:
In Python 3, if your file is defined as an UTF-8 encoded file (which it is, see file first line), you do not need to declare stuff as unicode anymore, thus IHMO, this should work:

```suggestion
                        "Erreur", "Mauvais mot de passe")
```

Reply:
And this could be applied everywhere... (but should first be tested...)